telemetry_components/src/main/java/com/newrelic/telemetry/SimpleSpanBatchSender.java

```diff
@@ -0,0 +1,32 @@
+package com.newrelic.telemetry;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.newrelic.telemetry.http.HttpPoster;
+import com.newrelic.telemetry.spans.SpanBatchSender;
+import com.newrelic.telemetry.spans.SpanBatchSenderBuilder;
+import java.time.Duration;
+
+public class SimpleSpanBatchSender {
+
+  public static SpanBatchSender build(String apiKey) {
+    return build(apiKey, Duration.ZERO);
+  }
+
+  public static SpanBatchSenderBuilder builder(String apiKey) {
+    return builder(apiKey);
+  }
+
+  public static SpanBatchSender build(String apiKey, Duration callTimeout) {
+    return builder(apiKey, callTimeout).build();
+  }
+
+  public static SpanBatchSenderBuilder builder(String apiKey, Duration callTimeout) {
+    HttpPoster http = new OkHttpPoster(callTimeout);
+    Gson gson = new GsonBuilder().create();
```

Comment:
does this do anything different than just `new Gson()` ?

Reply:
The docs say
```
 * <p>Use this builder to construct a {@link Gson} instance when you need to set configuration
 * options other than the default. For {@link Gson} with default configuration, it is simpler to
 * use {@code new Gson()}. {@code GsonBuilder} is best used by creating it, and then invoking its
 * various configuration methods, and finally calling create.</p>
 *
```
so yeah, I'll change it to the simpler constructor.  Good catch!  🏆